src/app/phast/sankey/sankey.component.ts

```diff
@@ -90,6 +90,9 @@ export class SankeyComponent implements OnInit {
       this.location = this.location.replace(/[\])}[{(]/g, '');
       this.location = this.location.replace(/#/g, "");
     }
+    console.log(this.modIndex);
+    console.log(this.location);
```

Comment:
A number of console.logs in various files. Also some stray comments around but they seem useful/explanatory.